main.py

```diff
@@ -104,6 +105,25 @@ def create_and_update_index(index_name, documents, fields_to_not_index):
     successCount, errors = helpers.bulk(es_client, documents, index=index_name)
     return successCount, errors
 
+def get_suggest_query(suggest_field):
+    return {
+        "field": suggest_field,
+        "size": 3,
+        "gram_size": 3,
+        "direct_generator": [
+            {"field": suggest_field, "suggest_mode": "missing"}
```

Comment:
Suggest mode options are "missing", "always", and "popular". Missing will only suggest if there are no results